tests/layer_tests/mo_python_api_tests/test_mo_convert_extensions.py

```diff
@@ -1,15 +1,21 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
please add this test running in the pre-commit. Now it isn't, as I understand, so it can be easily broken:)

Reply:
`@pytest.mark.precommit` mark should not work